quickget

```diff
@@ -906,7 +906,7 @@ function releases_nitrux() {
 }
 
 function releases_nixos() {
-    echo unstable 23.11 23.05
+    echo unstable 24.05 23.11 23.05
```

Comment:
```suggestion
    echo unstable $(web_pipe https://nix-channels.s3.amazonaws.com/?delimiter=/ | grep -o -P '(?<=<Key>nixos-)[0-9]+.[0-9]+(?=</Key>)' | sort -nr | head -n +2)
```

Try this instead. This will dynamically return the 2 newest releases, as well as unstable.
Plasma6 also should be added, as you mentioned. This PR does not appear to currently include it.

Reply:
roger, tested locally and added, ty.